manage-data/lifecycle/index-lifecycle-management/tutorial-general-content-with-data-streams.md

```diff
@@ -33,7 +33,26 @@ For more information, refer to [Check lifecycle progress](/manage-data/lifecycle
 
 ## Create an ingest pipeline to transform your general content [manage-general-content-with-data-streams-ingest]
 
-Create an ingest pipeline that uses the [`set` enrich processor](elasticsearch://reference/enrich-processor/set-processor.md) to add a `@timestamp` field:
+You can create an ingest pipeline that uses the [`set` enrich processor](elasticsearch://reference/enrich-processor/set-processor.md) to add a `@timestamp` field in Kibana or with the [create or update a pipeline](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-ingest-put-pipeline) API.
+
+::::{tab-set}
+:group: kibana-api
+:::{tab-item} {{kib}}
+:sync: kibana
+To add an ingest pipeline from {{kib}}, go to **Stack Management > Ingest Pipelines**. From the upper right, select **Create pipeline > New pipeline**.
```

Comment:
```suggestion
To add an ingest pipeline from {{kib}}, go to **Stack Management > Ingest Pipelines**. Select **Create pipeline > New pipeline**.
```

I'm not sure if this is still part of our guidelines, but for accessibility reasons we should try to avoid directional terms since they don't apply for anyone using a screen reader to access our docs.